src/components/Header.js

```diff
@@ -17,23 +19,33 @@ const useStyles = makeStyles({
 		marginLeft: "auto"
 	},
 	navbar: {
-		backgroundColor: "inherit"
+		backgroundColor: "inherit",
+		'& .MuiIconButton-label': {
+			color: "#282335"
+		}
 	}
 });
+
 const Header = () => {
 	const classes = useStyles();
+	const history = useHistory();
 	return (
 		<div className={classes.root}>
 			<AppBar position="static" className={classes.navbar}>
 				<Toolbar>
-					<Image src="https://via.placeholder.com/40" />
+					<Image src={logo} width-="40px" height="40px" />
 					<Typography variant="subtitle1">
-						{iconbuttons.map((button) => (
-							<IconButton variant="h1">{button.title}</IconButton>
+						{iconbuttons.map((button, i) => (
+							<IconButton variant="h1" onClick={() => {
+								history.push(button.url);
+							}} key={i}>{button.title}</IconButton>
 						))}
 					</Typography>
-					{authButtons.map((auth) => (
-						<Button text={auth.text} className={classes.authentication} />
+					{authButtons.map((auth, i) => (
```

Comment:
makes no difference in terms of performance..but you could consider destructuring at the map level just for readability.
```
authButtons.map(({url, text}, I)) => {
// rest of code
})
```